src/components/Header/Header.test.tsx

```diff
@@ -20,6 +20,14 @@ describe("Header", () => {
   beforeEach(() => {
     render(<Header />);
   });
+  it("renders a skip navigation", () => {
+    const skipNavigation = screen.getAllByRole("navigation")[0];
+    const { getByRole } = within(skipNavigation);
+
+    expect(skipNavigation).toBeInTheDocument();
+    expect(skipNavigation).toHaveAttribute("aria-label", "Skip Navigation");
+    expect(getByRole("list")).toBeInTheDocument();
```

Comment:
Why is the skip nav a list?

Reply:
It's a `nav` wrapper with an unordered list because there are two links. If there was only one, it would just be a `nav` wrapper and the anchor element without a list.